src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -228,6 +228,80 @@ class E
             comp.VerifyDiagnostics();
         }
 
+        [Fact, WorkItem(49263, "https://github.com/dotnet/roslyn/issues/49263")]
+        public void DeriveFromSelf()
```

Comment:
This test is an exact duplicate of `ToString_SelfReferentialBase` test in the same file.